### DISCUSSION TITLE
23.01.2024

### DISCUSSION THREAD

**Discussion by DGrillCoder** [grilldb97/legendary-space-zebra#22] [Show and tell]
Und wieder eine Woche vorbei. So langsam gehen mir die Ideen zum Programmieren aus.
Für kleinere Projekte fehlt mir irgendwie die Zeit, aber dennoch habe ich mich in ein neues Thema einarbeiten können: Apps
Dazu nutze ich das Kivy Package, welches sehr vielfältig ist. Ich habe ein tolles Tutorial auf Youtube gefunden: https://www.youtube.com/watch?v=l8Imtec4ReQ&t=5958s
Die Erklärungen sind super und leicht verständlich.

Ich habe mit einer einfachen App mit 'HelloWorld' gestartet. Wie man das so macht, wenn man in der Programmierung mit etwas Neuem startet ;)
Das Konvertieren von py in apk war dann nicht ganz so einfach. Als erstes habe ich von python-for-android gelesen. Irgendwie hat man da aber dann doch das Package buildozer benötigt. Dieses funktioniert aber aktuell nicht auf Windows. Darum habe ich WSL (windows subsystem for linux) installiert und damit ein System erstellt. Auch nach einigen Installationen von benötigten Modulen etc. hat das nicht so funktioniert, wie ich mir das vorgestellt habe. Nach weiterer Recherche bin ich dann auch eine tolle Anleitung gestoßen:  https://github.com/Spidy20/Python_To_APK
Das habe ich dann gleich ausprobiert mit Google Colab und es hat tatsächlich funktioniert.
Die apk habe ich heruntergeladen und auf mein Handy kopiert und installiert.
Nun habe ich meine erste eigene App auf meinem Handy.

Schon mit wenigen Elementen in Kivy kann man tolle Sachen machen.
Das hat echt potenzial und sieht jetzt schon nach einer Menge Spaß aus.
Fall ihr Lust auf eure eigene App habt, dann kann ich euch das YouTube Video nur empfehlen.

**Comment by NicoBaier123** [comment]
Coole Sache, planst du mit deiner App weiter zu machen?
